### PR TITLE
fix:release v0.2.3 bs bug

### PR DESCRIPTION
**Pull request by constwz**
### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

**Review by ruojunm — APPROVED** [review]

File: modular/blocksyncer/blocksyncer_options.go [hunk 1]
```diff
@@ -187,15 +187,17 @@ func (b *BlockSyncerModular) serve(ctx context.Context) {
 
 // enqueueNewBlocks enqueues new block heights onto the provided queue.
 func (b *BlockSyncerModular) enqueueNewBlocks(context context.Context, exportQueue types.HeightQueue, currHeight uint64) {
+	ticker := time.NewTicker(1500 * time.Millisecond)
 	// Enqueue upcoming heights
```
Comment (KeefeL) [inline modular/blocksyncer/blocksyncer_options.go:191]: defer ticker.Stop()

File: modular/blocksyncer/blocksyncer_options.go [hunk 2]
```diff
@@ -187,15 +187,17 @@ func (b *BlockSyncerModular) serve(ctx context.Context) {
...
 	for {
 		select {
 		case <-context.Done():
 			log.Infof("Receive cancel signal, enqueueNewBlocks routine will stop")
 			// close channel
 			close(exportQueue)
+			ticker.Stop()
```
Comment (KeefeL) [inline modular/blocksyncer/blocksyncer_options.go:198]: No need here, please use defer ticker.Stop()

File: modular/blocksyncer/blocksyncer_options.go [hunk 3]
```diff
@@ -187,6 +187,8 @@ func (b *BlockSyncerModular) serve(ctx context.Context) {
 
 // enqueueNewBlocks enqueues new block heights onto the provided queue.
 func (b *BlockSyncerModular) enqueueNewBlocks(context context.Context, exportQueue types.HeightQueue, currHeight uint64) {
+	ticker := time.NewTicker(1500 * time.Millisecond)
```
Comment (KeefeL) [inline modular/blocksyncer/blocksyncer_options.go:190]: Suggest 1500ms -> 500ms, the block interval is around 2s, if we use 1.5ms, that means in some case, you should wait around 1s to get the latest block, it's too slow.

File: modular/blocksyncer/blocksyncer_options.go [hunk 4]
```diff
@@ -206,17 +208,18 @@ func (b *BlockSyncerModular) enqueueNewBlocks(context context.Context, exportQue
 				}
 			}
 		}
-		time.Sleep(time.Second)
 	}
 }
 
 func (b *BlockSyncerModular) getLatestBlockHeight(ctx context.Context) {
+	ticker := time.NewTicker(1500 * time.Millisecond)
```
Comment (KeefeL) [inline modular/blocksyncer/blocksyncer_options.go:215]: The same here, 500ms is ok.

File: modular/blocksyncer/blocksyncer_options.go (hunk 3 above)
Comment (constwz) [reply modular/blocksyncer/blocksyncer_options.go:190]: This coroutine is to get the latest height to write to the queue, may write several blocks of height in one request, and then another coroutine to consume, consumption takes time, so we do not always need to get the latest height

**Review by KeefeL — APPROVED** [review]

File: modular/blocksyncer/blocksyncer_options.go [hunk 5]
```diff
@@ -286,17 +298,20 @@ func (b *BlockSyncerModular) fetchData(start, end uint64) {
...
 					continue
 				}
+				metrics.ChainRPCTime.Set(float64(time.Since(rpcStartTime).Milliseconds()))
 
 				events, err := b.parserCtx.Node.BlockResults(int64(height))
 				if err != nil {
 					log.Warnf("failed to get block results from node: %s", err)
 					continue
 				}
+				metrics.ChainRPCTime.Set(float64(time.Since(rpcStartTime).Milliseconds()))
```
Comment (ruojunm) [inline modular/blocksyncer/blocksyncer_options.go:314]: for this metrics, do we need an new rpcStartTime?

**Review by ruojunm — APPROVED** [review]

**Pull request merged**